force-app/main/default/lwc/accountModelSettings/accountModelSettings.js

```diff
@@ -1,20 +1,84 @@
-import { LightningElement, api, track } from "lwc";
+import { LightningElement, api, track, wire } from "lwc";
+
+import getAccountModelSettingsViewModel from "@salesforce/apex/AccountModelSettingsController.getAccountModelSettingsViewModel";
+
 import stgAccountModelSettingsTitle from "@salesforce/label/c.stgAccountModelSettingsTitle";
+import stgAccModelTitle from "@salesforce/label/c.stgAccModelTitle";
+import stgHelpAccountModel from "@salesforce/label/c.stgHelpAccountModel";
+import stgAdminAccountRecordType from "@salesforce/label/c.stgAdminAccountRecordType";
+import stgHelpAdminRecType from "@salesforce/label/c.stgHelpAdminRecType";
+import stgAccountRecordTypeSupportsHHAddress from "@salesforce/label/c.stgAccountRecordTypeSupportsHHAddress";
+import stgHelpHouseholdRecType from "@salesforce/label/c.stgHelpHouseholdRecType";
+import stgOptSelect from "@salesforce/label/c.stgOptSelect";
 
 export default class AccountModelSettings extends LightningElement {
     isEditMode = false;
     affordancesDisabledToggle = false;
 
+    @track accountModelSettingsViewModel;
+
+    labelReference = {
+        accountModelSettingsTitle: stgAccountModelSettingsTitle,
+        defaultAccountModelTitle: stgAccModelTitle,
+        defaultAccountModelDescription: stgHelpAccountModel,
+        adminAccountModelTitle: stgAdminAccountRecordType,
+        adminAccountModelDescription: stgHelpAdminRecType,
+        hhAccountModelTitle: stgAccountRecordTypeSupportsHHAddress,
+        hhAccountModelDescription: stgHelpHouseholdRecType,
+        comboboxPlaceholderText: stgOptSelect,
+        defaultAccountModelComboboxId: "defaultAccountModel",
+        adminAccountModelComboboxId: "adminAccountModel",
+        hhAccountModelComboboxId: "hhAccountModel",
```

Comment:
Oooh, elegant. These aren't technically labels. Maybe move these to another data structure? This is a great way to make it obvious that these are important.

Reply:
How about inputAttributeReference?  @xinnanlisf what are your thoughts?